modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.loader/03_oc_search_converter.js

```diff
@@ -102,8 +102,9 @@ class OpencastToPaellaConverter {
     return source;
   }
 
-  getStreamFromFlavour(episode, mainFlavour) {
-    var currentStream = { sources:{}, preview: '', content: mainFlavour };
+  getStreamFromFlavour(episode, flavour) {
+    var mainFlavour = flavour.split('/')[0],
+        currentStream = { sources:{}, preview: '', content: mainFlavour };
```

Comment:
@lkiesow , I concur with @staubesv that this removes the possibility to publish different types of tracks to Engage for different types of readers/players. At line 107, the track's published subflavor is ignored and lost. The pull removes the self documenting code about filtering by "*/delivery" and retains assumptions about track flavors and what the "flavour.split('/')[0]" symbolizes. 

That "flavour.split('/')[0]" is more cryptic than before. It would help to add a comment regarding assumptions about flavors and that tracks of any published subflavor are grouped here by their main flavor. i.e. "presenter/abc", "presenter/xyz".

Ideally, at some future point, it'd be nice to configure the flavors that the Paella player accepts. If the configuration is missing or empty, take all subflavors and group them by their mainFlavor ;)